reviews/graduation-jaeger.md

```diff
@@ -0,0 +1,74 @@
+# Jaeger Graduation Application
+
+Jaeger was open sourced in April, 2017 and joined the CNCF in September, 2017. The project has grown significantly over time. As of Oct-30-2018 ([devstats][devstats]):
+
+  * 6185 stars on GitHub
+  * 878 forks
+  * 224 authors of commits and pull requests
+  * 350 issue creators  
+
+Jaeger fulfills all CNCF incubating and graduation criteria. The following application links to the required information to become a graduated project.
+
+## Document that it is being used successfully in production by at least three independent end users which, in the TOC’s judgement, are of adequate quality and scope.
+
+Known end users include: Alibaba Cloud, Base CRM (acquired by ZenDesk), Circonus, ContaAzul, FarmersEdge, GrafanaLabs, Massachusetts Open Cloud, Northwestern Mutual, Nets, Stagemonitor, RiksTV, SeatGeek, SpotHero, Vistar Media, Uber, Weave, Weaveworks, Zenly.
```

Comment:
Please add some further detail on at least three of these production use cases to enable the TOC to get a sense of the "quality and scope" aspect of the requirement.  For example, in which specific product/project Jaeger is used (rather than just the company name), size of deployment (both in terms of infrastructure size and volume of usage/users), how long it's been in production for, etc.  Anything that you feel will help the TOC to evaluate.

Reply:
@yurishkuro here is ContaAzul's usage details:

ContaAzul company currently develops an ERP for small and medium business in Brazil  (https://contaazul.com ) where this ERP is basically one legacy monolith with a bunch of microservices around it and in some situations those microservices interact with each other independently. Currently we have around 60 microservices deployed to production with 20 of them already instrumented and the rest being prioritized.

We deploy our microservices to a self-maintained Kubernetes cluster on AWS. Each Kubernetes node holds a Jaeger agent responsible for collecting spans from its node pods. Our storage is ElasticSearch on AWS ElasticSearch Service, avoiding to maintain the cluster ourselves, and it currently have 2B searchable documents related to Jarger spans

Jaeger is helping us to increase observability since Feb 2018 and is praised by our developers.

Main problems solved: Observability, bottlenecks lookup

Feel free to add this to the graduation proposal!

Reply:
@bboreham is there something you can add from Weaveworks experience?

Reply:
added several user examples